packages/venia-concept/src/components/MiniCart/product.css

```diff
@@ -92,7 +70,7 @@
     right: -24px;
     top: -7px;
     bottom: -7px;
-    background-color: rgb(var(--venia-teal));
-    opacity: 0.1;
+    background-color: rgb(var(--venia-grey));
+    opacity: 0.5;
```

Comment:
This class should be renamed to `mask` probably. `modal` isn't accurate anymore.